### PR TITLE
feat: add delete stock from stock db functionality

### PR DESCRIPTION
**Pull request by Rae-Kwon**
I have added the delete stock from the stock db functionality with this commit!!! Would like some suggestions on whether having the stock db is useful because at the moment any user can add or delete from the pool!!!!

### CODE REVIEW

File: src/routes/stocks/delete-stock.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      res.status(400).json({
+        message: 'Please input a symbol that is in the database',
+      });
+      return;
+    }
+
+    try {
+      await Stock.deleteOne({ symbol });
+    } catch (err) {
+      const msg = 'Something went wrong saving stock to database';
```
Comment (JessieFrance) [inline src/routes/stocks/delete-stock.ts:36]: We should eventually change this line to say something about deleting the stock from the database.

**Review by JessieFrance — APPROVED** [review]
Great work Richard! I tested adding a stock, and then tested this method using Postman and it works. 

I think you ask a great question on this route about how anyone can delete a stock. If we decide to keep the `user-stock` model, then we could get in trouble if someone deletes stocks that the `user-stock` model references. One solution would be for us to add middleware that makes this route only available to some privileged user, and then delete stocks that don't belong to anyone. I think this route is particularly useful in that case for doing database maintenance with a script (as we don't have a lot of space in the MongoDB Atlas sandbox plan). If people decide they still want to use the `user-stock` model, then we should also create a `delete` route there as well.  In any case, thank you for your work on this!

File: src/app.ts [hunk 2]
```diff
@@ -1,13 +1,14 @@
 import express from 'express';
 import 'express-async-errors';
 import cors from 'cors';
-import { json } from 'body-parser';
+import { json, urlencoded } from 'body-parser';
```
Comment (DanielFleace1) [inline src/app.ts:4]: Great work on this code. Just as a learning moment for myself. Why did you add urlencoded. I was able to remove it and still successful use the delete route. Is it for future use or what am I missing! Thanks!

**Review by DanielFleace1 — APPROVED** [review]
Great work. I was able to successfully use your route and it handled errors correctly. 

As Jesse said you raised a great question. If we want to get this app completely  ironed out I think we probably need one more team meeting with a set agenda to align our end gamification  goal. This would make it easier to discuss the trade offs Jesse describes.
 
 On the other hand, hope i'm not the only one out of the loop feeling like that haha!
 I vote to approve this route & we can change that in the future if needed!

**Pull request merged**